graphene_django/tests/urls_pretty.py

```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import url
+from django.urls import re_path
 
 from ..views import GraphQLView
 from .schema_view import schema
 
-urlpatterns = [url(r"^graphql", GraphQLView.as_view(schema=schema, pretty=True))]
+urlpatterns = [re_path(r"^graphql", GraphQLView.as_view(schema=schema, pretty=True))]
```

Comment:
I think it's better to use `path` with a string non-regex URL.

Reply:
The same comment applies for all `url` - `re_path` changes.

Reply:
This change was a bit more annoying than anticipated, but everything should be good now.